File: SofiForce.Web/ClientApp/src/app/core/Models/ListModels/ClientPreferredTimeListModel.ts
export interface ClientPreferredTimeListModel {
    preferredId: number;
    clientId: number | null;
    preferredOperationId: number | null;
    weekDayId: number | null;
    fromTime: string | null;
    toTime: string | null;
    weekDayName: string;
    preferredOperationName: string;
}

File: SofiForce.Models/Models/ListModels/ClientPreferredTimeListModel.cs
using System;

namespace Models
{
    public class ClientPreferredTimeListModel
    {
        public long? PreferredId { get; set; }
        public int? ClientId { get; set; }
        public int? PreferredOperationId { get; set; }
        public int? WeekDayId { get; set; }
        public TimeSpan? FromTime { get; set; }
        public TimeSpan? ToTime { get; set; }
        public string WeekDayName { get; set; }
        public string PreferredOperationName { get; set; }

    }
}
